Formula/jolie.rb

```diff
@@ -8,11 +8,7 @@ class Jolie < Formula
 
   def install
     system "java", "-jar", "jolie-#{version}.jar", "-jh", libexec, "-jl", bin
```

Comment:
If they exist, please use long form arguments here for readability

Reply:
Done

Reply:
updated accordingly